js/ui/handler/scroll_zoom.js

```diff
@@ -97,6 +97,7 @@ ScrollZoom.prototype = {
     },
 
     _zoom: function (delta) {
+        if (delta === 0) return;
```

Comment:
yeah, I don't have a strong opinion either way, I'm just trying to figure out / learn if that actually makes sense to do this

I'm thinking that since there will always be a cutoff, whether it's epsilon or 0, some values that could be skipped might not be. I think epsilon would be more arbitrary than 0 because it would be just a blind guess. What makes `1 * epsilon` a better cutoff than `2 * epsilon` or `0.5 * epsilon`? What makes `1 * epsilon` better than `0 * epsilon`?

It would make sense to use a non-zero value if we had real examples, but I don't know if a blind guess for a problem we haven't seen actually improves anything. It seems like it just makes everything a tiny bit more complicated